pom.xml

```diff
@@ -1177,6 +1180,16 @@ under the License.
       </dependency>
       <dependency>
         <groupId>org.webjars.bower</groupId>
+        <artifactId>angular-translate</artifactId>
+        <version>${angular-translate.version}</version>
+      </dependency>
+      <dependency>
```

Comment:
This dependency is also available as `org.webjars.bower:angular-translate-loader-partial`, with same version as `angular-translate`.

I would replace

```
<dependency>
       <groupId>org.webjars</groupId>
      <artifactId>angular-translate-loader-partial</artifactId>
       <version>${angular-translate-loader-partial.version}</version>
     </dependency>
```

with

```
<dependency>
    <groupId>org.webjars.bower</groupId>
    <artifactId>angular-translate-loader-partial</artifactId>
    <version>${angular-translate.version}</version>
</dependency>
```

and adjust the `index.html` above accordingly.